src/components/Header/Header.tsx

```diff
@@ -1,38 +1,78 @@
 import React from 'react';
 import './Header.scss';
+import cn from 'classnames';
+import { useProductStore } from '../../store/productStore';
 import { Link } from 'react-router-dom';
 
+import logo from '../../assets/icons/logo.svg';
+import like from '../../assets/icons/heart.svg';
+import cart from '../../assets/icons/cart-icon.svg';
+import menu from '../../assets/icons/menu.svg';
+import close from '../../assets/icons/close.svg';
+
 export const Header: React.FC = () => {
+  const isMenuOpen = useProductStore(state => state.isMenuOpen);
+  const toggleMenu = useProductStore(state => state.toggleMenu);
+
   return (
     <header className="header">
       <div className="header__container">
-        <a className="header__logo" href="#home"></a>
-
-        <nav className="header__nav">
-          <Link className="header__nav-link" to="home">
-            HOME
-          </Link>
-          <Link className="header__nav-link" to="phones">
-            PHONES
-          </Link>
-          <Link className="header__nav-link" to="tablets">
-            TABLETS
-          </Link>
-          <Link className="header__nav-link" to="accessories">
-            ACCESSORIES
-          </Link>
+        <Link to="">
+          <img src={logo} alt="NiceGadgetsLogo" className="header__logo" />
+        </Link>
+
+        <nav
+          className={cn('nav', {
+            'nav--open': isMenuOpen,
+          })}
+        >
+          <div className="nav__container">
+            <Link className="nav__container-link" to="#home">
+              HOME
+            </Link>
+
+            <Link className="nav__container-link" to="#phones">
+              PHONES
+            </Link>
+
+            <Link className="nav__container-link" to="tablets">
+              TABLETS
+            </Link>
+
+            <Link className="nav__container-link" to="accessories">
+              ACCESSORIES
+            </Link>

```

Comment:
here you should probably use `to="home"`, not `"#home"`. as well as `/accessories`, and others. but please check if this will work in your code

Reply:
Now fix it